test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -677,28 +842,167 @@ void ProtoConverter::visit(Block const& _x)
 		m_output << "{}\n";
 }
 
-void ProtoConverter::visit(Function const& _x)
+void ProtoConverter::visit(SpecialBlock const& _x)
+{
+	m_numVarsPerScope.push(0);
+	m_output << "{\n";
+	visit(_x.var());
+	if (_x.statements_size() > 0)
+	{
+		for (auto const& st: _x.statements())
+			visit(st);
+	}
+	m_numLiveVars -= m_numVarsPerScope.top();
+	m_numVarsPerScope.pop();
+	m_output << "}\n";
+}
+
+template <class T>
+void ProtoConverter::createFunctionStub(T const& _x, int _numInParams, int _numOutParams, NumFunctionReturns _type)
```

Comment:
Does it create the function definition stub or the function call stub?

Reply:
done